README.md

```diff
@@ -1,61 +1,27 @@
 React Firebase
 ==============
 
-React bindings for [Firebase](https://www.firebase.com).
+React bindings for [Firebase](https://firebase.google.com).
 
 ## Installation
 
-React Firebase requires **React 0.14 or later.**
+React Firebase requires **React 0.14 and Firebase 3 or later.**
 
 ```
 npm install --save react-firebase
 ```
 
 ## Usage
```

Comment:
The documentation is quite heavy and detailed, so I would suggest having a simple table of contents at the top, or/and give a simple example before diving in to Static Properties, arguments etc.